README.md

```diff
@@ -33,3 +33,25 @@ WORK IN PROGRESS
     1. Trigger OpenWhisk to process data
     2. Send newly processed data to dashboard
 4. Dashboard displays reports
+
+## Example
+
+To use the [sample python script](example/gh_traffic.py), you will need to [create a personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with public_repo access:
+![personal_access_token_settings](images/personal_access_token_settings.png "Personal Access Token Settings")
+
+Then, run [`example/gh_traffic.py`](example/gh_traffic.py), entering your personal access token and repo path when asked:
+
+```shell
+$ $(which python) example/gh_traffic.py
+GitHub personal access token: ****************************************
+GitHub repo (username/reponame): IBM/github-traffic-stats-collector
```

Comment:
In this case this organization_name/reponame. Maybe you can say anything after github.com, or have them enter the full URL